package.json

```
@@ -0,0 +1,6 @@
+{
+  "name": "bqml",
+  "dependencies": {
+    "@dataform/core": "2.9.0"
```

```suggestion
    "@dataform/core": "3.0.0-beta.4"
```

Just so it's up to date as possible.